Project-I-Social-Card/social-card/src/App.js

```diff
@@ -5,10 +5,12 @@ import CardContainer from "./components/CardComponents/CardContainer";
 
 const App = () => {
   return (
-    <div className = 'appCard'>
+    <React.Fragment>
       <HeaderContainer />
```

Comment:
very nice way to just wrap the card container with the anchor tag, quick note, there is no www in ractjs.org, also we could have done this with a click handler, not sure if Ryan went over this yet though.